docs/en/connector-v2/sink/Druid.md

```diff
@@ -0,0 +1,101 @@
+# Druid
+
+> Druid sink connector
+
+## Description
```

Comment:
Please add the `Key Features` reference https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/connector-v2/sink/Clickhouse.md